cmd/telegraf/agent.conf

```
@@ -94,3 +94,12 @@
   ## stateful plugins on termination of Telegraf. If the file exists on start,
   ## the state in the file will be restored for the plugins.
   # statefile = ""
+
```

I did not suggest to remove that line, please revert.
```suggestion

  ## The interval of time at which the state is peristed into the statefile.
```